gsm/gsm-64bit.patch

```diff
@@ -0,0 +1,15 @@
+--- gsm-1.0-pl12/inc/private.h.64bit	2006-04-26 21:17:01.000000000 +0200
++++ gsm-1.0-pl12/inc/private.h	2007-05-11 00:25:23.000000000 +0200
+@@ -10,10 +10,10 @@
+ #define	PRIVATE_H
+ 
+ typedef short			word;		/* 16 bit signed int	*/
```

Comment:
I think it would be better to refactor this patch in terms of using `stdint.h` types like `int16_t` and `uint16_t` and so on.  This makes the patch more robust and stops encoding toolchain-specific assumptions into the types.

Reply:
I tried to do that refactoring on patch but the CI failed :/

> _https://github.com/wolfi-dev/os/actions/runs/5413419072/jobs/9838942778?pr=3172#step:7:288_